core-bundle/composer.json

```diff
@@ -96,6 +95,7 @@
         "symfony/process": "^5.3",
         "symfony/routing": "^5.3",
         "symfony/security-bundle": "^5.3",
+        "symfony/string": "^5.3",
```

Comment:
As the core bundle is using  lots of`mb_*` functions, don't we need to add `"symfony/polyfill-mbstring": "^1.0"` here as well?

Reply:
You are absolutely right. Added in 17882bf720ca06945124c88d8b827ed2bcb2f06a.